src/components/binding/bindResults.html

```diff
@@ -1,31 +1,36 @@
 <div ng-if="!ctrl.error && !(ctrl.binding | isBindingFailed)">
   <div ng-if="ctrl.binding && !(ctrl.binding | isBindingReady)" class="results-status">
-    <span class="spinner spinner-sm" aria-hidden="true"></span>
+    <span class="fa fa-clock-o text-muted" aria-hidden="true"></span>
```

Comment:
The spinner was part of the UX design. I like this better but should be approved by @openshift/team-ux-review

Reply:
I've been working with @ncameronbritt and co. on this.  The clock is the approved solution.